.github/actions/action.yml

```diff
@@ -10,6 +10,7 @@ runs:
   # Tell Conan to look for or create its build folder (.conan) in the repository's root directory
   env:
     CONAN_USER_HOME: $GITHUB_WORKSPACE
+    PACKMAN_ROOT: $GITHUB_WORKSPACE/packman-repo
```

Comment:
Is this env variable used?

Reply:
Oops, that's an error. Let me fix quickly.